pkgs/by-name/st/streamcontroller/package.nix

```diff
@@ -16,18 +16,24 @@
 stdenv.mkDerivation rec {
   pname = "streamcontroller";
 
-  version = "1.5.0-beta.7";
+  version = "1.5.0-beta.8";
   # We have to hardcode revision because upstream often create multiple releases for the same version number.
-  # This is the commit hash that maps to 1.5.0-beta.7 released on 2024-11-20
-  rev = "45b5bc72f617c5aea306450d6592da66ade53568";
+  # This is the commit hash that maps to 1.5.0-beta.8 released on 2025-03-12
+  rev = "de11d84afac7873044568606a8468c78d57aceda";
```

Comment:
Do we need to expose rev as an env?

Reply:
Nope, I can move it outside of the derivation